### DISCUSSION TITLE
will twidere-x also shutdown?

### DISCUSSION THREAD

**Discussion by Adithyan007hack** [TwidereProject/TwidereX-Android#561] [General] [answered]
I have heard Twitter is shutting down third-party clients
will twidere-x also shut down?
Harpy already had released there shut down notice https://github.com/robertodoering/harpy/discussions/884

**Comment by Tlaster** [comment, marked as answer]
Not yet👀, and we still plan to support fediverse platforms like Mastodon or Misskey in the future.
